services/intersection-api/api/src/main/resources/application.yaml

```diff
@@ -49,12 +49,12 @@ conflict:
 spring:
   data:
     mongodb:
-      database: ConflictMonitor
+      database: ${CM_DATABASE_NAME:CV}
```

Comment:
Note: This being hard-coded was resulting in failure to communicate with mongo after switching over to using the service provided by `jpo-utils`